src/Library/Library.elm

```diff
@@ -24,24 +37,29 @@ popTop library =
         x :: xs ->
             ( Just x, xs )
 
+
 popBottom : Library -> ( Maybe Card, Library )
 popBottom library =
     case library of
         [] ->
             ( Nothing, [] )
+
         xs ->
             let
                 card =
                     reverse xs |> head
```

Comment:
I would extract `reverse xs` into a variable so the computation is not run twice.